packages/mobile/src/consumerIncentives/reducer.ts

```diff
@@ -0,0 +1,35 @@
+import { createAction, createReducer } from '@reduxjs/toolkit'
+import { SuperchargePendingReward } from 'src/consumerIncentives/types'
+
+export interface State {
+  loading: boolean
+  error: boolean
+}
+
+export const initialState = {
+  loading: false,
+  error: false,
+}
+
+export const claimRewards = createAction<SuperchargePendingReward[]>('SUPERCHARGE/CLAIM_REWARDS')
+export const claimRewardsSuccess = createAction('SUPERCHARGE/CLAIM_REWARDS_SUCCESS')
+export const claimRewardsFailure = createAction('SUPERCHARGE/CLAIM_REWARDS_FAILURE')
+
+export const reducer = createReducer(initialState, (builder) => {
```

Comment:
We should switch to `createSlice` for new code.
See example in [src/i18n/slice.ts](https://github.com/valora-inc/wallet/blob/main/packages/mobile/src/i18n/slice.ts)

Fine to do in a separate PR given the time sensitivity of this.